sdk/monitor/Azure.Monitor.Query/src/Models/MetricsClientAudience.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Monitor.Query
+{
+    /// <summary> Cloud audiences available for MetricsClient. </summary>
+    public readonly partial struct MetricsClientAudience : IEquatable<MetricsClientAudience>
+    {
+        private readonly string _value;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MetricsClientAudience"/> object.
+        /// </summary>
+        /// <param name="value">The Microsoft Entra audience to use when forming authorization scopes. For the language service, this value corresponds to a URL that identifies the Azure cloud where the resource is located. For more information: <see href="https://learn.microsoft.com/en-us/azure/azure-government/documentation-government-manage-oms" />.</param>
```

Comment:
I don't think this link is helpful, so let's remove it.
```suggestion
        /// <param name="value">The Microsoft Entra audience to use when forming authorization scopes. This value corresponds to a URL that identifies the Azure cloud where the resource is located.</param>
```